.gitignore

```diff
@@ -1,6 +1,7 @@
 .DS_Store
 .awcache
 .cache
+.cache-loader
```

Comment:
This is an old directory that you can `rm -rf`, it doesn’t need to be ignored anymore.

Reply:
Maybe worth leaving there for a minute since it was recently deprecated?  Was surprised to have 1500 changed files on a PR today, and had no reference for what `.cache-loader` referred to.

Reply:
Or just shouting out to #dev slack that we should remove

Reply:
In retrospect, I think leaving the directory in the ignore list for now is probably the most workable solution 👍